sdk/core/Azure.Core/tests/public/ModelSerializationTests/ModelXTests.cs

```diff
@@ -2,7 +2,9 @@
 // Licensed under the MIT License.
 
 using System;
```

Comment:
There are a few models in the test projects that don't have a Tests file inheriting from ModelJsonTests or ModelTests

We should either remove this models if they aren't needed or add the test class to validate those models.